controlpanel/frontend/jinja2/parameter-create.html

```diff
@@ -0,0 +1,108 @@
+{% from "fieldset/macro.html" import govukFieldset %}
+{% from "radios/macro.html" import govukRadios %}
+
+{% extends "base.html" %}
+
+{% set page_title = "Create a secret parameter for your webapp or airflow task" %}
+{% set legend -%}
+  Create a secret parameter for your webapp or airflow task
+{%- endset %}
+
+{% set page_name = "parameters" %}
+
+{% block content %}
+  <h1 class="govuk-heading-xl">{{ page_title }}</h1>
+
+  {% if form.error %}
+    <div class="error">{{ form.error }}</div>
+  {% endif %}
+
+  <form action="{{ url('create-parameter') }}" method="post">
+    {{ csrf_input }}
+    {% call govukFieldset({ "legend": { "html": legend|safe } }) %}
+      {{
+        govukInput({
+          "label": {
+            "text": "Key",
+            "classes": "govuk-label--m",
+          },
+          "classes": "govuk-!-width-two-thirds",
+          "hint": {
+            "text": '60 chars max, only letters, numbers and underscores'
+          },
+          "name": "key",
+          "attributes": {
+            "pattern": "[a-zA-Z0-9_]{1,60}",
+            "maxlength": "60",
+          },
+          "value": form.key.value(),
+          "errorMessage": { "html": form.key.errors|join("<br>") } if form.key.errors else {}
+        })
+      }}
+      {{
+        govukInput({
+          "label": {
+            "text": "Value",
+            "classes": "govuk-label--m",
+          },
+          "classes": "govuk-!-width-two-thirds",
+          "hint": {
+            "text": 'String. Any characters'
+          },
+          "name": "value",
+          "attributes": {
+          },
+          "value": form.value.value(),
+          "errorMessage": { "html": form.value.errors|join("<br>") } if form.value.errors else {}
+        })
+      }}
+      {{
+        govukInput({
+          "label": {
+            "text": "Role name",
+            "classes": "govuk-label--m",
+          },
+          "classes": "govuk-!-width-two-thirds",
+          "hint": {
+            "text": '60 chars max, only lowercase letters and underscores'
+          },
+          "name": "role_name",
+          "attributes": {
+            "pattern": "[a-z_]{1,60}",
+            "maxlength": "60",
+          },
+          "value": form.role_name.value(),
+          "errorMessage": { "html": form.role_name.errors|join("<br>") } if form.role_name.errors else {}
+        })
+      }}
+      {{
+        govukRadios({
+          "label": {
+            "text": "Airflow or Web app",
+            "classes": "govuk-label--m",
+          },
+          "classes": "govuk-!-width-two-thirds",
+          "hint": {
+            "text": 'Will the secret param be used in Airflow or in a Web app?'
+          },
+          "name": "used_by",
+          "attributes": {},
+          "items": [
+            {
+              "value": "airflow",
+              "text": "Airflow",
+            },
+            {
+              "value": "webapp",
+              "text": "Web app",
+            }
```

Comment:
Hardcoded choices again. These should at least be passed as template context variables.

Reply:
Have added to template context